js/client/modules/@arangodb/testutils/process-utils.js

```diff
@@ -67,6 +67,7 @@ const termSignal = 15;
 
 let tcpdump;
 let assertLines = [];
+let logCounter = 0;
```

Comment:
It seems that this variable is only incremented, but its value is not read back anywhere. I think this can be removed.